cache.js

```diff
@@ -0,0 +1,8 @@
+const redis = require('redis');
+const client = redis.createClient();
+
+client.on("error", function(err) {
+    console.log("Error " + err);
```

Comment:
I was under the impression that Redis would always try to reconnect automatically if the errir was handled. I confess I don't have a lot of experience with Redis in general.